src/file_reader_h5.py

```diff
@@ -116,9 +134,10 @@ def open_h5_files_and_return_file_data(file_paths):
                                     data_preparation_and_conversion[2])
                                 new_velocity = {'velocity': velocity}
                                 dataset_from_file.append(new_velocity)
+                                single_dataset["velocity"] = list(new_velocity)
```

Comment:
Hier habe ich fälschlicher weise new_velocity übergeben, das ist aber ja ein objekt, daher kann die DB damit nichts anfangen. Vorher hatte ich velocity übergeben, da kam aber das problem, dass das scheinbar an dieser stelle ab und an "None" war. quick and dirty wäre hier "is not None" aber irgendwas stimmt da nicht so ganz mit dem algorithmus dahinter.

Reply:
@lk24dx

Reply:
@felixkru das sollte jetzt gefixt sein, velocity kann nicht mehr none sein, das lag daran, dass teilweise ein timestamp nicht gelesen werden konnt, da alles groß